frontend/src/tabs/zoom.pug

```diff
@@ -15,11 +15,25 @@
       v-bind:avgsize="info.avgCommitSize",
       v-bind:mergegroup="info.isMergeGroup")
 
+  .zoom__toggle-commit-message-body(v-if="info.user.commits.length")
+    button(v-if="expandedCommitMessagesNum === 0" v-on:click="toggleAllCommitMessagesBody(true)") show all commit messages body
+    button(v-else v-on:click="toggleAllCommitMessagesBody(false)") hide all commit messages body
+
   .zoom__day(v-for="day in info.user.commits")
     template(v-if="day.insertions > 0")
       h3(v-if="filterTimeFrame === 'week'") Week of {{ day.date }}
       h3(v-else) {{ day.date }}
       ul
-        li(v-for="slice in day.commitResults")
-          a(v-bind:href="getSliceLink(slice)", target="_blank") {{ slice.messageTitle }}
+        li(v-for="slice in day.commitResults").commit-message.active
+          a(v-bind:href="getSliceLink(slice)" target="_blank") {{ slice.messageTitle }}
           span &nbsp; ({{ slice.insertions }} lines)
+          a(
+            v-if="slice.messageBody.length > 0",
+            v-on:click="updateExpandedCommitMessagesNum",
```

Comment:
This is being used as a counter, so instead of `Num`, naming it `Count` is more appropriate